department-of-veterans-affairs/va.gov-cms
# VACMS-10132: Select fields missing labels in CMS

## Description

Closes #10132.

## Testing done
Tested locally on ddev.

## Screenshots


## QA steps
Select any existing piece of VAMC System Operating Status content. In the "System-wide alerts" section and "Facility operating statuses" section click Show row weights. The weight number should be visible now. Also when you inspect the selects you should see a label with a visually hidden span. 

I used `/node/28390/edit` to work in.

### Select Team for PR review
